public/login_success.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Login Success</title>
+</head>
+
+<body>
+  <noscript>
+    You need to enable JavaScript to be able to login!
+  </noscript>
+  Login Success!
+  <script>
+    document.body.onload = function () {
+      var loggedInUser = { data: { username: 'dude' } };
+      window.opener.postMessage(
+        {
+          payload: loggedInUser,
+          status: 'success'
+        },
+        window.opener.location
+      );
+    };
+  </script>
+</body>
+
+</html>
```

Comment:
new line here

Reply:
this file will be gone when it's final state